src/java.desktop/share/native/libfreetype/src/base/ftcolor.c

```diff
@@ -0,0 +1,156 @@
+/****************************************************************************
+ *
+ * ftcolor.c
+ *
+ *   FreeType's glyph color management (body).
+ *
+ * Copyright (C) 2018-2022 by
+ * David Turner, Robert Wilhelm, and Werner Lemberg.
+ *
```

Comment:
Per the above copyright appears neither you nor JetBrains have ownership of this file.
Therefore per the OCA you cannot contribute this file to OpenJDK.
If it is really needed, then as a pre-cursor step, it will need to be integrated as a 3rd party file as part of the freetype 3rd party library sources, and that will have to be done by an Oracle employee.

This is a blocker for moving forward.

Reply:
Yes, I believe it is really needed. Extracting colored outlines wouldn't work without it.

Reply:
OK, I will submit a PR to take care of this soon.

Reply:
Thank you.

Reply:
Pushed under bug https://bugs.openjdk.org/browse/JDK-8298974

Reply:
You need to remove this from your change and resync to get the JDK's copy